authui/src/authflowv2/botprotection/cloudflareTurnstile.ts

```diff
@@ -0,0 +1,29 @@
+import { Controller } from "@hotwired/stimulus";
+
+declare global {
+  interface Window {
+    turnstile: any;
+  }
+}
+
+export class CloudflareTurnstileController extends Controller {
+  static values = {
+    siteKey: { type: String },
+    theme: { type: String },
+  };
+
+  declare siteKeyValue: string;
+  declare themeValue: string;
+
+  connect() {
+    window.turnstile.ready(() => {
+      window.turnstile.render(this.element, {
+        sitekey: this.siteKeyValue,
```

Comment:
Experiment and document the configuration https://developers.cloudflare.com/turnstile/get-started/client-side-rendering/#disable-implicit-rendering

For example, when will the expire callback be called? What should we do to handle the expire callback? What does the refresh expire option do?